projects/transfer-journey/src/lib/transfer-journey.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+import { MakeTransferJourneyConfigService } from './make-transfer-journey-config.service';
+
+@Component({
+  selector: 'lib-transfer-journey',
```

Comment:
All of the components have `bb-` prefix and here is `lib-`.
Do we need prefixes at all?
If so, should they not be aligned?

Reply:
yes I removed the selector name as it is used in the route